R/signal.R

```diff
@@ -1,75 +1,58 @@
-#' Signal experimental or superseded features
+#' Signal other life cycle changes
```

Comment:
Should we keep the experimental badge for this function until we have actually used it in real applications?

Reply:
Yeah, good call.